ffbuild.sh

```diff
@@ -194,7 +194,7 @@ detect_arch_switch $MINGVER
 TARGET=$BASE/target/$MINGVER/
 WORK=$BASE/work/$MINGVER/
 PMTEST="$BASE/.pacman-$MINGVER-installed"
-PYINST=python3
+PYINST=python
```

Comment:
https://github.com/fontforge/fontforge/actions/runs/12917757007/job/36024837540?pr=5531

That build complains about `mingw-w64-x86_64-python-pip`? Is this really needed after applying Syy?